lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,21 +214,21 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx, view_as_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True, False): (48, 6, 13),
-        ('no_overrides', 2, True, False): (135, 6, 84),
-        ('no_overrides', 3, True, False): (480, 6, 335),
-        ('ccx', 1, True, False): (48, 6, 13),
-        ('ccx', 2, True, False): (135, 6, 84),
-        ('ccx', 3, True, False): (480, 6, 335),
+        ('no_overrides', 1, True, False): (49, 6, 13),
```

Comment:
@andy-armstrong @mattdrayer If you see the `released_languages() common/lang_pref/api.py#16` method then you can see that this method is already caching the `dark_lang` model object So I think It will not effect the performance if we call `released_languages()` method in `navigation.html`, It just fetch the data from model in first hit and It will cache it.

**Now the question is then, why is it required to increment theses values in this above test ?**  If you look into this test above then you will see, test is clearing all system caches before measuring so we are not dependent on cache in this test.
Please correct me if I am missing something.